### ISSUE TITLE
Refactor novicell.visible.js

### ISSUE THREAD

**Issue by shp-novicell** [Novicell/novicell-frontend#137]
Use new syntax: https://github.com/Novicell/novicell-frontend/issues/111

**Comment by magnify** [comment]
Refactored the script here: https://github.com/magnify/ncstyleguide/blob/master/source/js/novicell.visible.js

Got an example/test page on the way.

This is a new script, so there is no target for at pull request. @Dan9boi should we create a new repo at https://github.com/Novicell and handle code review there?

**Comment by magnify** [comment]
Example here: https://magnify.github.io/ncstyleguide/

**Comment by Dan9boi** [comment]
Very cool, though i don't get the demo?
We use this script when lazyloading images, maybe a demo with that would make more sense?

I'm not sure if we need the left/right check, but what if we did a second parameter - `threshold` for checking if element is within fx. 500px from the top/bottom of the screen?

I've got a hold of the script @marklonquist did and it is very similar to that:
```javascript
function isScrolledIntoView(el) {
	var elemTop = el.getBoundingClientRect().top;
	var elemBottom = el.getBoundingClientRect().bottom;

	return (elemBottom >= 0) && (elemTop <= window.innerHeight);
}
```

I'll make a new repo for you to push and move the discussion to that: https://github.com/Novicell/novicell-visible
